pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -292,7 +294,61 @@ func (m *kubeGenericRuntimeManager) KillPod(pod *api.Pod, runningPod kubecontain
 // GetPodStatus retrieves the status of the pod, including the
 // information of all containers in the pod that are visble in Runtime.
 func (m *kubeGenericRuntimeManager) GetPodStatus(uid kubetypes.UID, name, namespace string) (*kubecontainer.PodStatus, error) {
-	return nil, fmt.Errorf("not implemented")
+	// Now we retain restart count of container as a docker label. Each time a container
```

Comment:
Container's restart count is get by this function, so those comments are put here. We may move those comments later to `SyncPod` where consumes restart count.